src/components/settings/settings-assigned-users/settings-assigned-users.js

```diff
@@ -95,7 +95,7 @@ const SettingsAssignedUsers = ({
               </Button>
             )}
           >
-            <span>[find-user-plugin is not available]</span>
+            <span><FormattedMessage id="ui-eholdings.settings.assignedUsers.noPluginFindUser" /></span>
```

Comment:
Could you please check here we have `noPluginFindUser`, but in translation file `no-plugin-find-user`?